model/riscv_sys_regs.sail

```diff
@@ -157,17 +157,19 @@ val ext_veto_disable_C : unit -> bool effect {rreg}
 /* We currently only support dynamic changes for the C extension. */
 function legalize_misa(m : Misa, v : xlenbits) -> Misa = {
   if   sys_enable_writable_misa ()
-  then { /* Handle modifications to C. */
-         let  v = Mk_Misa(v);
-         /* Suppress changing C if nextPC would become misaligned or an extension vetoes or C was disabled at boot (i.e. not supported). */
-         let m =
-           if   (v.C() == 0b0 & (nextPC[1] == bitone | ext_veto_disable_C())) | ~(sys_enable_rvc())
-           then m
-           else update_C(m, v.C());
-         /* Handle updates for F/D. */
-         if   ~(sys_enable_fdext()) | (v.D() == 0b1 & v.F() == 0b0)
+  then { 
+         let v = Mk_Misa(v);
+         /* Suppress disabling C & keep misa unchanged if nextPC would become misaligned or an extension vetoes*/ 
+         if  v.C() == 0b0 & (nextPC[1] == bitone | ext_veto_disable_C())
          then m
-         else update_D(update_F(m, v.F()), v.D())
+         else {
+           /* Handle updates for C */
+           let m = if  ~(sys_enable_rvc()) then m else update_C(m, v.C());
```

Comment:
```suggestion
           let m = if ~(sys_enable_rvc()) then m else update_C(m, v.C());
```
The extra spacing is only to pad `if` to the same length as `then`, which is only wanted when the `then` is on the next line, not when it's a one-liner.